maintenance/image-options/private-registry-regular.md

```diff
@@ -25,7 +25,7 @@ A **private registery** requires an **image pull secret**.
 
 An **image pull secret** provides authentication for an image registry; this allows you to control access to certain images or give access to higher pull rate limits (like with DockerHub).
 
-An **image path** is a directory in a public or private registry that contains images required to install {{site.prodname}}.
+An **image path** is a directory in registry that contains images required to install {{site.prodname}}.
```

Comment:
minor: `directory in registry` should be `directory in a registry`

Reply:
Fixed.